test/k8sT/hubble.go

```diff
@@ -257,14 +269,32 @@ var _ = Describe("K8sHubbleTest", func() {
 				helpers.CurlFail(fmt.Sprintf("http://%s/public", app1ClusterIP)))
 			res.ExpectSuccess("%q cannot curl clusterIP %q", appPods[helpers.App2], app1ClusterIP)
 
-			// In case a node was temporarily unavailable, hubble-relay will
-			// reconnect once it receives a new request. Therefore we retry
-			// in a 5 second interval.
-			hubbleObserveUntilMatch(ciliumPodK8s1, fmt.Sprintf(
-				"--server %s --last 1 --type l7 --from-pod %s/%s --to-namespace %s --to-label %s --protocol http",
-				hubbleRelayAddress, namespaceForTest, appPods[helpers.App2], namespaceForTest, app1Labels),
-				`{$.Type}`, "L7",
-				helpers.MidCommandTimeout)
+			flows := getFlowsFromRelay(fmt.Sprintf(
+				"--last 1 --type l7 --from-pod %s/%s --to-namespace %s --to-label %s --protocol http",
+				namespaceForTest, appPods[helpers.App2], namespaceForTest, app1Labels))
+			Expect(flows).NotTo(BeEmpty())
+		})
+
+		It("Test FQDN Policy with Relay", func() {
+			fqdnProxyPolicy := helpers.ManifestGet(kubectl.BasePath(), "fqdn-proxy-policy.yaml")
+			fqdnTarget := "vagrant-cache.ci.cilium.io"
+
+			_, err := kubectl.CiliumPolicyAction(
+				namespaceForTest, fqdnProxyPolicy,
+				helpers.KubectlApply, helpers.HelperTimeout)
+			Expect(err).To(BeNil(), "Cannot install fqdn proxy policy")
+			defer kubectl.CiliumPolicyAction(namespaceForTest, fqdnProxyPolicy,
+				helpers.KubectlDelete, helpers.HelperTimeout)
+
+			res := kubectl.ExecPodCmd(namespaceForTest, appPods[helpers.App2],
+				helpers.CurlFail(fmt.Sprintf("http://%s", fqdnTarget)))
+			res.ExpectSuccess("%q cannot curl fqdn target %q", appPods[helpers.App2], fqdnTarget)
+
+			flows := getFlowsFromRelay(fmt.Sprintf(
+				"--last 1 --type trace:from-endpoint --from-pod %s/%s --to-fqdn %s",
```

Comment:
Question unrelated to the PR: are the subtypes documented somewhere? Looks `hubble --help` shows only the types (i.e. `l7 agent policy-verdict drop debug capture trace`).

Reply:
I don't think they are :/

Reply:
Added an issue to track this at https://github.com/cilium/hubble/issues/462